src/auth/ldap_krb/KrbAuthorizeHandler.cpp

```diff
@@ -0,0 +1,69 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2017 Daniel Oliveira <doliveira@suse.com>
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ *
+ */
+
+/* Include order and names:
+ * a) Immediate related header
+ * b) C libraries (if any),
+ * c) C++ libraries,
+ * d) Other support libraries
+ * e) Other project's support libraries
+ *
+ * Within each section the includes should
+ * be ordered alphabetically.
+ */
+
+#include "KrbAuthorizeHandler.hpp"
+
+#include "common/debug.h"
+
+#define dout_subsys ceph_subsys_auth 
+
+bool KrbAuthorizeHandler::verify_authorizer(CephContext* ceph_ctx, 
+                                            KeyStore* keys, 
+                                            bufferlist& authorizer_data,  
+                                            bufferlist& authorizer_reply, 
+                                            EntityName& entity_name, 
+                                            uint64_t& global_id,  
+                                            AuthCapsInfo& caps_info, 
+                                            CryptoKey& session_key, 
+                                            uint64_t* auid, 
+                                            std::unique_ptr<
+                                              AuthAuthorizerChallenge>* challenge)
+{
+  constexpr auto SUBSYSTEM_ID(0);
```

Comment:
this is being used as the log level, not a subsystem id. Why not just explicitly setting the log level on `ldout` instead?